### PR TITLE
Update <head> to include more metadata

### PR DESCRIPTION
**Pull request by lipmaneth**
@ivelin Currently, the website does not share a preview image nor does it contain metatags for SEO and google. This PR is intended to do the following:

- Add a preview image that's visible when shared on social or via SMS. See #3 for associated image.
- Add a title tag usable by Google shown in search results.
- Add a meta description usable by Google shown in search results.

The developer already had some basic metadata, which I did not delete. I am unsure if his code should remain or if any of it should be removed. I played it safe and left it.

### CODE REVIEW

**Comment by lipmaneth** [comment]
deploy preview https://raw.githack.com/GuardianUI/landing-page/848635b95d74dc18aae844e7d0c8d50b32142e13/index.html

File: index.html [hunk 1]
```diff
@@ -25,6 +25,24 @@
...
+        <meta property="og:description" content="Provide users with a safer and more robust experience. Testing, monitoring, and alerts in one platform.">
+        <meta property="og:image" content="assets/images/Testing and Monitoring (1).png">
+
+        <!-- Twitter -->
+        <meta property="twitter:card" content="summary_large_image">
+        <meta property="twitter:url" content="https://www.guardianui.com/">
+        <meta property="twitter:title" content="GuardianUI - Frontend testing and monitoring for web3 apps">
+        <meta property="twitter:description" content="Provide users with a safer and more robust experience. Testing, monitoring, and alerts in one platform.">
+        <meta property="twitter:image" content="assets/images/Testing and Monitoring (1).png">
         <meta
```
Comment (ivelin) [inline index.html:39]: Repetitive meta description. Drop one.

**Review by ivelin — CHANGES_REQUESTED** [review]
Nice work. See the duplicate tag comment.

**Review by ivelin — APPROVED** [review]
well done

**Pull request merged**